src/Angor/Avalonia/AngorApp.Desktop/AngorApp.Desktop.csproj

```diff
@@ -1,27 +1,31 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <OutputType>WinExe</OutputType>
-    <!--If you are willing to use Windows/MacOS native APIs you will need to create 3 projects.
-    One for Windows with net9.0-windows TFM, one for MacOS with net9.0-macos and one with net9.0 TFM for Linux.-->
-    <TargetFramework>net8.0</TargetFramework>
-    <Nullable>enable</Nullable>
-    <BuiltInComInteropSupport>true</BuiltInComInteropSupport>
-  </PropertyGroup>
+    <PropertyGroup>
+        <OutputType>WinExe</OutputType>
+        <!--If you are willing to use Windows/MacOS native APIs you will need to create 3 projects.
+        One for Windows with net9.0-windows TFM, one for MacOS with net9.0-macos and one with net9.0 TFM for Linux.-->
+        <TargetFramework>net9.0</TargetFramework>
```

Comment:
Oh, is there a reason why you downgraded? I'm using .NET 9 just to see how if there's a noticeable performance gain for the application.

Reply:
yes it seems teh code did not work in dotnet9 when publishing (it would work locally) so there is an issue with deployment process in dotnet9 and blazor.

There is a branch to attempt the upgrade without disturbing the release
https://github.com/block-core/angor/pull/216